RealmSwift-swift2.2/Tests/RealmTests.swift

```diff
@@ -255,6 +255,29 @@ class RealmTests: TestCase {
         }
         XCTAssertEqual(try! Realm().objects(SwiftStringObject.self).count, 0)
     }
+    
+    func testThrowsWrite() {
+        enum TestError: ErrorType { case Exception }
```

Comment:
Sorry, this is the up-to-date line: https://github.com/realm/realm-cocoa/blob/78c8690be82048fd21858cc3c8128efceec12360/RealmSwift-swift2.2/Tests/RealmTests.swift#L262